ReactiveCocoa/Swift/SignalProducer.swift

```diff
@@ -316,8 +316,28 @@ extension SignalProducerType {
 	///
 	/// Returns a Disposable which can be used to interrupt the work associated
 	/// with the Signal, and prevent any future callbacks from being invoked.
-	public func start(error error: (E -> ())? = nil, completed: (() -> ())? = nil, interrupted: (() -> ())? = nil, next: (T -> ())? = nil) -> Disposable {
-		return start(Event.sink(next: next, error: error, completed: completed, interrupted: interrupted))
+	public func startWithNext(error error: (E -> ())? = nil, completed: (() -> ())? = nil, interrupted: (() -> ())? = nil, next: (T -> ())? = nil) -> Disposable {
```

Comment:
After thinking about it a bit more, I tend to agree it might be the least confusing option. Time will tell if it would be too cumbersome to use.

So if we agree on this, I'll go ahead and revert my last commit and go back to the original suggestion.